scanomatic/models/validators/compile_instructions_model.py

```diff
@@ -0,0 +1,65 @@
+import os
+from typing import Literal, Union
+
+from scanomatic.data_processing.calibration import get_active_cccs
+from scanomatic.io.fixtures import Fixtures
+from scanomatic.io.paths import Paths
+from scanomatic.models.compile_project_model import (
+    FIXTURE,
+    CompileInstructionsModel,
+    CompileInstructionsModelFields,
+)
+
+ValidationResult = Union[Literal[True], CompileInstructionsModelFields]
+
+
+def validate_images(
+    model: CompileInstructionsModel,
+) -> ValidationResult:
+    if model.images:
+        return True
+    else:
+        return CompileInstructionsModelFields.images
+
+
+def validate_path(
+    model: CompileInstructionsModel,
+) -> ValidationResult:
+    basename = os.path.basename(model.path)
+    dirname = os.path.dirname(model.path)
+    if (
+        model.path != dirname
+        and os.path.isdir(dirname)
+        and os.path.abspath(dirname) == dirname
+        and basename
+    ):
+        return True
+    return CompileInstructionsModelFields.path
+
+
+def validate_fixture(model: CompileInstructionsModel) -> ValidationResult:
+    if model.fixture_type is FIXTURE.Local:
+        if os.path.isfile(
+            os.path.join(
+                model.path,
+                Paths().experiment_local_fixturename,
```

Comment:
A shame to have to instantiate the `Paths` on each check, but I guess it doesn't happen too often.

Reply:
yeah it really shouldn't be a class I think, but a module